.github/workflows/test.yml

```diff
@@ -104,6 +104,51 @@ jobs:
       - name: "Run integration tests"
         run: docker run -t --rm --privileged test-integration
 
+  test-integration-ipv6:
```

Comment:
We have too many tests now, can we just merge test-integration-ipv6 to test-integration?

Reply:
If not, tes-integration-ipv6 should be just executed for Ubuntu 22.04 and containerd v1.7

Reply:
Previously, it was merged. However, when using the `--network host` option in Docker to run integration tests, many non-IPv6 tests failed. Therefore, I separated it and ran it independently.